libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -236,7 +236,12 @@ export class DropdownComponent implements AfterViewInit, OnDestroy, ControlValue
     event.preventDefault();
   }
 
-  /* Utility that makes it easier to set styles on card element 
+  onTouchStart(event: TouchEvent): void {
+    // Prevent touchmove propgation to allow for scroll on page with pull-to-refresh;
+    event.stopPropagation();
+  }
```

Comment:
Consider replacing this with the following snippet. 

I think it should result in identical behavior, and it uses a similar pattern to how we have done it for the keydown events in this component (bottom of this file). As an added bonus we should not need to add anything to the template.
 
```
@HostListener('touchstart', ['$event']) _onTouchStart(event: TouchEvent) {
  event.stopPropagation();
}
```

Reply:
Fixed